orm/support.adoc

```diff
@@ -5,14 +5,8 @@
 [[supported-versions]]
 == Supported Products pass:[<i class="icon-user-md icon-fixed-width icon-2x"></i>]
 
-Hibernate ORM is a fully supported component of
+Hibernate ORM is a fully supported component of the https://www.redhat.com/en/products/runtimes[Red Hat Runtimes] offering.
 
-* http://www.redhat.com/products/jbossenterprisemiddleware/application-platform/[Red Hat JBoss Enterprise Application Platform]
-* http://www.redhat.com/products/jbossenterprisemiddleware/web-server/[Red Hat JBoss Enterprise Web Server]
-* http://www.redhat.com/products/jbossenterprisemiddleware/portal/[Red Hat JBoss Portal ]
-* http://www.redhat.com/products/jbossenterprisemiddleware/data-grid/[Red Hat JBoss Data Grid]
-* http://www.redhat.com/products/jbossenterprisemiddleware/business-rules/[Red Hat JBoss BRMS]
-* http://www.redhat.com/products/jbossenterprisemiddleware/soa/[Red Hat SOA Platform]
-* http://www.redhat.com/products/jbossenterprisemiddleware/developer-studio/[Red Hat JBoss Developer Studio]
+Using JBoss EAP, Quarkus, Open Liberty or Spring Boot? They are all part of Red Hat Runtimes.
```

Comment:
I wasn't sure we should keep this list? Most of the pages redirect to the Middleware homepage now. And I thought support really made sense as part of the Red Hat Runtimes offering.

But happy to reconsider if someone has concerns (and ideas about how to word it)